### DISCUSSION TITLE
fooocus always produce symmetrical images

### DISCUSSION THREAD

**Discussion by lmst2** [lllyasviel/Fooocus#2305] [Q&A]
I am new to ai drawings. I am using run_anime settings, I have tested for bunch of different prumpt. and I found that it would always produce symmetrical images, especially in the first few itrations it would start with a symmetrical chaos. dose that relat to some sarting method? I want to know, what caused this, and what can I do to avoid this?
For example: what I want is a shop with one mirror wall, but I get this. symmetrical two equipments on both sides, whitch is wired in real life.
<img width="406" alt="Snipaste_Snipper - Snipaste_2024-02-19_21-12-22" src="https://github.com/lllyasviel/Fooocus/assets/29498103/063ba469-e03d-49d9-83f0-2c78b1c69a18">

**Comment by poor7** [comment]
Default settings. No Styles. Prompt: same
`{
"Prompt": "a hair cut shop with dianagional lines of mirrors and chairs, small windows cusing a dark environment, light on the cillings, rellections on the ground",
"Negative Prompt": "",
"Fooocus V2 Expansion": "",
"Styles": "[]",
"Performance": "Speed",
"Resolution": "(1152, 896)",
"Sharpness": 2,
"Guidance Scale": 4,
"ADM Guidance": "(1.5, 0.8, 0.3)",
"Base Model": "juggernautXL_v8Rundiffusion.safetensors",
"Refiner Model": "None",
"Refiner Switch": 0.5,
"Sampler": "dpmpp_2m_sde_gpu",
"Scheduler": "karras",
"Seed": 1234,
"LoRA 1": "sd_xl_offset_example-lora_1.0.safetensors : 0.1",
"Version": "v2.1.865"
}`
![2024-02-20_04-04-45_5934](https://github.com/lllyasviel/Fooocus/assets/154618384/668ca0ea-08d7-466a-84e1-b0738a54b7f3)

**Reply by lmst2** [reply]
thats looks aswome. can I know what caused symmetric? styles? models? sampler? or other settings? how to have a detailed control on what I want to see? lots of questions haha. it will be great if you can answer me that. thanks!

**Reply by poor7** [reply]
It's difficult to answer your questions, because... you haven't provided any technical information about your process, just a small image.

**Reply by lmst2** [reply]
I used the anime preset with styles "Fooocus V2, Fooocus Enhance, SAI Anime", the model i used I believe is 'animaPencilXL_v100'. if anything else you want me to provide, please tell me and I'll find and tell you.

**Reply by poor7** [reply]
The model animaPencilXL_v100 reacts poorly to some prompts. The aspect ratio you're using suggests something narrow. Additional styles dilute your keywords (prompts).
